Updated Token Hook API and OIDC API with correct aud and sub reserved… (#2234)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/token-hook/index.md

   ]
}
```

> > **Note:** Attempting to add an element within an array that doesn't exist or specifying an invalid index results in the entire PATCH failing and errors logged in the token hooks events.

Reviewer: ```suggestion
> **Note:** If you attempt to add an element within an array that doesn't exist or specify an invalid index, the entire PATCH will fail and errors will be logged in the token hooks events.
```

### Sample Response to Replace an Existing Claim

You can modify existing custom claims or OIDC standard profile claims, such as `birthdate` and `locale`. You can't, however, modify any system-specific claims, such as `iss` or `ver`, and you can't modify a claim that isn't currently part of the token in the request payload. Attempting to modify a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.

> For the list of access token reserved claims that you can't modify, see [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims).

Reviewer: For cross-reference text to links, the style guide says to word it like this: See...for...

Similarly, you can replace elements in an array by specifying the array name and the valid index of the element that you want to replace in the path.

### Sample Response to Modify Token Lifetime

You can modify how long the access and ID tokens are valid by specifying the `lifetime` in seconds. The `lifetime` value must be a minimum of five minutes (300 seconds) and a maximum of 24 hours (86,400 seconds).

```json
{
   "commands":
     [
        {
            "type": "com.okta.identity.patch",
            "value": [
                 {
                     "op": "replace",
                     "path": "/token/lifetime/expiration",
                     "value": 36000
                  }
             ]
         },
         {
             "type": "com.okta.access.patch",
             "value": [
                  {
                     "op": "replace",
                     "path": "/token/lifetime/expiration",
                     "value": 36000
                   }
              ]
          }
      ]
}
```

### Sample Response to Remove Token Claims

> You can remove existing custom claims or OIDC standard profile claims, such as `birthdate` or `locale`. You can't, however, remove any system-specific claims, such as `iss` or `ver`, and you can't remove a claim that isn't currently part of the token in the request payload. Attempting to remove a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.

Reviewer: ```suggestion
You can remove existing custom claims or OIDC standard profile claims, such as `birthdate` or `locale`. You can't, however, remove any system-specific claims, such as `iss` or `ver`, and you can't remove a claim that isn't currently part of the token in the request payload. If you attempt to remove a system-specific claim or use an invalid operation, the entire PATCH will fail and errors will be logged in the token hooks events.
```

> For the list of access token reserved claims that you can't remove, see [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims).

Reviewer: ```suggestion
See [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims) for the list of access token reserved claims that you can't remove.
```

You can also use the `remove` operation to remove members from existing JSON objects and elements from existing arrays. For example, you have an array that contains the user's preferred airports, and you want to remove an airport from the array.

Existing target JSON object:

```json
{
   "preferred_airports":[
      "sjc",
      "lax",
      "sfo",
      "oak"
   ]
}
```

> You can remove the element from the array by specifying the array name followed by the index of the element that you want to remove. You don't need to specify a value for the `remove` operation. But, you can specify `null` as the value if you want.

Reviewer: ```suggestion
You can remove the element from the array by specifying the array name followed by the index of the element that you want to remove. You don't need to specify a value for the `remove` operation, but you can specify `null` as the value if you want.
```
